elliottlib/cli/__main__.py

```diff
@@ -362,6 +378,11 @@ def repair_bugs(runtime, advisory, auto, id, original_state, new_state, noop, de
         if bug.status in original_state:
             changed_bug_count += 1
             elliottlib.bzutil.set_state(bug, new_state, noop=noop)
+        if close_placeholder and "Placeholder" in bug.summary:
+            changed_bug_count += 1
+            elliottlib.bzutil.set_state(bug, "CLOSED", noop=noop)
+        if comment and not noop:
+            bug.setstatus(comment=comment, private=False)
```

Comment:
This line can be moved to the scope where set_state is called, because we don't need to add a comment to the placeholder bug.